packages/app/app/App.js

```diff
@@ -68,6 +72,26 @@ class App extends React.PureComponent {
     this.updateConnectivityStatus(navigator.onLine);
     window.addEventListener('online', () => this.updateConnectivityStatus(true));
     window.addEventListener('offline', () => this.updateConnectivityStatus(false));
+
+    ipcRenderer.on('navigate-back', this.handleBack);
```

Comment:
Could you define these events in `@nuclear/core` in the IpcEvents enum?

Reply:
And a better place for this is the IpcContainer

Reply:
Moved it to IpcContainer and defined the events in IpcEvents